### ISSUE TITLE
Cannot Import MSFS 2020 aircraft into Blender.

### ISSUE THREAD

**Issue by Shevymax** [AsoboStudio/glTF-Blender-IO-MSFS#230]
### Current Behavior

Trying to import aircraft into Blender from MSFS 2020 with no success.

I get an error "Extension MSFS_texture_dds is not available on this addon version" 

Reviewed and applied solutions provided in #151 and #154, this is my first time trying this application so I have no history or experience with any of these applications.

### Expected Behavior

Expected to see a MSFS 2020 glTF option on the Import dropdown menu as per numerous YouTube videos
Actually see: glTF 2.0 (.glb/ .glTF) only

Expected fix in #154 to work, already had it implemented so no next steps available....?
Expected to be able to import an aircraft to make custom skin, tried multiple aircraft types with no success.

### Steps To Reproduce

1. Install addon as per instructions
2. Enable addon via instructions and Enabled Importer Extensions
3. Choose File / Import / gltf
4. Choose gltf file, Import fails. (Tried planes in the Community Folder also)

### Environment

```markdown
- OS: Windows 10
- Blender: Version 3.1
- glTF-Blender-IO-MSFS: Version 1.1.6
```

### Anything else?

N/A

**Comment by J-1775** [comment]
Not sure if this answers your question (I'm a noob, too):
**⚠️ NOTE:1 This plugin cannot import glTF files that have been built into a Microsoft Flight Simulator package through the sim's Package Builder.**
That would usually apply to packages in both community and official folder.

**Comment by Shevymax** [comment]
Thank you for taking the time to answer my question, then are there any
planes that can be imported and custom skins made? A commmon one on YouTube
is the A320 Neo, now mind you that’s a year or two ago and ASOBO may have
put a stop to it because I cannot import it.

Also, do you have the MSFS glTF as an option in your import drop down menu?
If so how?

**Comment by 4evercool** [comment]
> Thank you for taking the time to answer my question, then are there any planes that can be imported and custom skins made? A commmon one on YouTube is the A320 Neo, now mind you that’s a year or two ago and ASOBO may have put a stop to it because I cannot import it. Also, do you have the MSFS glTF as an option in your import drop down menu? If so how?
>
Asobo plugin has never been able to do what you want. There was another plugin built by FBW team member pepperoni, but it has been discontinued long ago.

**Comment by J-1775** [comment]
> Also, do you have the MSFS glTF as an option in your import drop down menu? If so how?

No, on import I see only the glTF 2.0. But when I open the toolbox ('N') there's is a tab that reads **Multi-Export glTF 2.0**. And when the plugin is activated in material properties you'll see «MSFS material params» which you can fold out.
I work with scenery addons, not aircraft, can't say much about that. But I think the idea is that rather than opening existing MSFS aircraft in Blender you create your own models or take 3D objects from libraries like sketchfab to import and work on.

**Comment by Shevymax** [comment]
Awesome, thanks for the update!
